opencanary/logger.py

```diff
@@ -164,13 +165,21 @@ def error(self, data):
 
     def log(self, logdata, retry=True):
         logdata = self.sanitizeLog(logdata)
+        
         # Log only if not in ignorelist
         notify = True
         if 'src_host' in logdata:
             for ip in self.ignorelist:
                 if check_ip(logdata['src_host'], ip) == True:
                     notify = False
                     break
+
+        if 'logtype' in logdata:
```

Comment:
```
if 'logtype' in logdata and logdata['logtype'] in self.logtype_ignorelist:
```

Reply:
change done on logtyp test condition and add ip_ to ignorelist at line 172